### PR TITLE
docs: Fix Readme of how to run popular docker hub images script

### LINKED ISSUE #417: Fix Readme of popular images
Fix readme of how to run popular docker hub images

### PR DESCRIPTION
**Pull request by GabyCT**
Change of the name of the script.

Fixes #417

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/docker_popular_images/README [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 To run these tests you just do
 
-bats popular-containers-images.bats
+bats popular_docker_images.bats
```
Comment (grahamwhaley) [inline integration/docker_popular_images/README]: obviously correct :-)
but, whilst here, as I checked - I see we have the `popular_docker_images.bats` file living under the `docker_popular_images` folder. How would you feel about unifying them to be the same word order whilst fixing this doc? :-)

**Comment by GabyCT** [comment]
@grahamwhaley changes applied

**Comment by GabyCT** [comment]
@iphutch could you please take a look of this pr? thanks

**Comment by devimc** [comment]
lgtm

**Comment by chavafg** [comment]
lgtm

**Pull request merged**